config/oracle_jre.yml

```diff
@@ -20,7 +20,7 @@
 # e.g.  repository_root: "http://example.com/oracle-jre/{platform}/{architecture}"
 ---
 jre:
-  version: 1.8.0_+
+  version: 1.8.0_12+
```

Comment:
I haven't tested this but I don't think the qualifier is the issue, its the ordering/sorting. 1.8.0_101 is sorted before 1.8.0_121 as its using naive sorting. 

So whilst 1.8.0_1+ does work, I don't think it sorts properly. So from {1.8.0_101,1.8.0_121} gives you u101 and not u121. 

I'll try to confirm if I get time.